modules/vrtcalBidAdapter.js

```diff
@@ -10,16 +10,25 @@ export const spec = {
   },
   buildRequests: function (bidRequests) {
     const requests = bidRequests.map(function (bid) {
-      const params = {
+      let floor = 0;
+
+      if (typeof bid.getFloor === 'function') {
```

Comment:
can you use the isFn in the utils here?

Reply:
> can you use the isFn in the utils here?

Thanks Chris...Updated to utilize isFn and isPlainObject.